README.md

```diff
@@ -1,17 +1,32 @@
 # Halogen Template
 
-This is a template for starting a fresh project with the [Halogen](https://github.com/slamdata/purescript-halogen) library for writing declarative, type-safe user interfaces.
+### Quick Start
+```
+git clone https://github.com/purescript-halogen/purescript-halogen-template.git halogen-project
+cd halogen-project
+npm install -g purescript spago parcel
+npm run build
+npm run serve
+```
+
+### Introduction
+
+This is a template for starting a fresh project with the [Halogen](https://github.com/purescript-halogen/purescript-halogen) library for writing declarative, type-safe user interfaces. This template mimics the introductory example of the [official React Hooks documentation](https://reactjs.org/docs/hooks-intro.html).
```

Comment:
Do you feel that it is necessary to note the similarity to the official React Hooks documentation? Unless there's something in that repository that affects how you should approach this one I'd rather not introduce React.

Reply:
Probably not. Will remove.